playbooks/group_vars/all/public.yml

```diff
@@ -2,7 +2,7 @@
 
 # ANSIBLE #####################################################################
 
-ansible_python_interpreter: /usr/bin/python3
+ansible_python_interpreter: /usr/bin/python
```

Comment:
I think we want to keep using python3 by default, but just use python2 for the MongoDB servers.